UnityProject/Assets/Scripts/Items/Implants/ImplantLimb.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using HealthV2;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)